### PR TITLE
Borer can't use its power while dead

### LINKED ISSUE #24623: Borers can use their fear stun post-death
1) Open menu to pick fear target.
2) Get murdered.
3) Selected target, target still gets stunned?

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Fixes #24623

### CODE REVIEW

File: code/game/gamemodes/miniantags/borer/borer.dm [hunk 1]
```diff
@@ -468,7 +468,7 @@ var/total_borer_hosts_needed = 10
 	var/mob/living/carbon/M = input(src,"Who do you wish to dominate?") in null|choices
 
 
-	if(!M || !src)
+	if(!M || !src || stat || (world.time - used_dominate < 150))
```
Comment (RemieRichards) [inline code/game/gamemodes/miniantags/borer/borer.dm]: ```|| victim```

Comment (AnturK) [inline code/game/gamemodes/miniantags/borer/borer.dm]: Please use defines for stat checks.

Comment (AnturK) [inline code/game/gamemodes/miniantags/borer/borer.dm]: Kor please.

Comment (KorPhaeron) [inline code/game/gamemodes/miniantags/borer/borer.dm]: I just got home

**Comment by KorPhaeron** [comment]
@AnturK 

Should be done unless I fucked something up in the web editor

**Pull request merged**